src/structure/3dmol/assignBonds.ts

```diff
@@ -87,6 +87,7 @@ function bondLength(elem: string): number {
 // return true if atom1 and atom2 are probably bonded to each other
 // based on distance alone
 function areConnected(atom1: AtomSpec, atom2: AtomSpec) {
+    if (atom1.elem === "X" || atom2.elem === "X") return false;
```

Comment:
```suggestion
    if (atom1.elem === "X" || atom2.elem === "X") {
        return false;
    }
```

should fix the prettier issue